dtests/src/test/java/io/snappydata/hydra/cluster/startDualModeCluster.bt

```diff
@@ -0,0 +1,5 @@
+io/snappydata/hydra/cluster/startDualModeCluster.conf
```

Comment:
The name startDualModeCluster.conf is already used for starting snappy cluster and smart mode cluster and purpose for both is different. We should think on renaming one of these as it may be confusing later, when we refactor the tests.